src/main/java/com/csbank/v1/BankAccount.java

```diff
@@ -1,6 +1,46 @@
 package com.csbank.v1;
 
 public class BankAccount {
+    private int balance;
+
+    public BankAccount(int balance) {
+        this.balance = balance;
+    }
+
+    public int getBalance() {
+        return balance;
+    }
+
+    public boolean deposit(int deposit) {
+        if (deposit > 0){
+            balance = balance + deposit;
+        } else {
+            System.out.println("Invalid input.");
+            return false;
+        }
+        return true;
+    }
+
+    public boolean withdraw(int withdraw) {
```

Comment:
hm withdraw should not return false.  If this happens in real bank, you'll be upset with the bank.  It remove the amount from your account, but you're not getting anything back other than success/fail status